api/services/DownloadStatsService.js

```diff
@@ -1,6 +1,9 @@
 var _ = require('lodash');
 var Promise = require('bluebird');
 var dateFormat = require('dateformat');
+var CSV = require('csv-js');
```

Comment:
Probably removed it when removing other packages I added but didn't need eventually